dart/lib/src/event_processor/deduplication_event_processor.dart

```diff
@@ -27,6 +27,10 @@ class DeduplicationEventProcessor extends EventProcessor {
 
   @override
   FutureOr<SentryEvent?> apply(SentryEvent event, {hint}) {
+    if (event is SentryTransaction) {
+      return event;
+    }
```

Comment:
since Dart does not have method overloading, we can't do the same as Java for `EventProcessor`, we'd need different methods and call either one or the other, so right now skipping the processor if its a transaction

Reply:
JS does event.type == 'transaction'